src/jit/compiler.h

```diff
@@ -465,6 +469,19 @@ class LclVarDsc
         assert(_lvRegNum == reg);
     }
 
+    __declspec(property(get = GetIsParam, put = SetIsParam)) char lvIsParam;
+
+    char GetIsParam() const
+    {
+        return _lvIsParam;
+    }
+
+    void SetIsParam(char isParam)
+    {
+        _lvIsParam = isParam;
+        if (!isParam) lvIsImplicitByRef = 0;
```

Comment:
In what scenario would we be setting `lvIsParam` to false (and thus have to reset `lvIsImplicitByRef`?